src/agent/src/tracer.rs

```diff
@@ -81,6 +84,8 @@ pub fn setup_tracing(name: &'static str, logger: &Logger, _agent_cfg: &AgentConf
 
     tracing::subscriber::set_global_default(subscriber)?;
 
+    global::set_text_map_propagator(TraceContextPropagator::new());
```

Comment:
If we're still going to allow for ["isolated" agent tracing](https://github.com/kata-containers/kata-containers/pull/1937), I think this is where we need to check if `TraceType::Collated` has been requested.

See: https://github.com/kata-containers/kata-containers/pull/1937#issuecomment-855812254

Reply:
From the discussion we had during the Architecture Committee meetings seems that "isolated" is not the goal, at least not in the first interactions of this, right?